solution/backend/file_manager/admin.py

```diff
@@ -54,16 +55,11 @@ class UploadedFileAdmin(BaseAdmin):
         "subjects": lambda: Subject.objects.all()
     }
 
-    # Will remove any characters from file names we do not want in it.
-    # Commas in file names causes issues in chrome on downloads since we rename the file.
-    def clean_file_name(self, name):
-        name = name.replace(',', '')
-        return name
-
     def save_model(self, request, obj, form, change):
         path = form.cleaned_data.get("file_path")
         if path:
-            obj.file_name = self.clean_file_name(path._name)
+            file_name, extension = path._name.split('.')
+            obj.file_name = f"{slugify(file_name).replace('-', ' ')}.{extension}"
```

Comment:
Although you use slugify you are also replacing the '-' with ' ' so a simple test would be good to have here that we expect the file names to be cleaned up in this way. Thanks

Reply:
removed replace to simplify the code.